src/utils/conda_environment_file.rs

```diff
@@ -118,33 +118,20 @@ fn parse_dependencies(deps: Vec<CondaEnvDep>) -> miette::Result<ParsedDependenci
             CondaEnvDep::Pip { pip } => pip_deps.extend(
                 pip.into_iter()
                     .map(|mut dep| {
+                        // FIXME: newer versions of uv should be able to deal with git URL directly
                         let re = Regex::new(r"/([^/]+)\.git").unwrap();
                         if let Some(caps) = re.captures(dep.as_str()) {
                             let name= caps.get(1).unwrap().as_str().to_string();
                             tracing::warn!("The dependency '{}' is a git repository, as that is not available in pixi we'll try to install it as a package with the name: {}", dep, name);
-                            dep = name;
+                            dep = format!("{name} @ {dep}");
                         }
-                        let req = pep508_rs::Requirement::from_str(&dep).into_diagnostic()?;
-                        let name = PyPiPackageName::from_normalized(req.name.clone());
-                        let requirement = PyPiRequirement::from(req);
-                        Ok((name, requirement))
+                        pep508_rs::Requirement::from_str(&dep).into_diagnostic()
                     })
                     .collect::<miette::Result<Vec<_>>>()?,
             ),
         }
     }
 
-    if !pip_deps.is_empty()
```

Comment:
I don't think adding pip in the environment is needed when importing from a conda environment file. Unless I am missing something?

Reply:
It shouldn't but it might be expected to exist, if you move from conda to pixi.

But now we support editables this might indeed be better to loosen up.

Reply:
My thoughts exactly :-)